src/networkcreator.js

```diff
@@ -1055,6 +1057,60 @@ class NetworkCreator extends EventEmitter {
     return needToCreateGeth(this.config) ? gethService : undefined
   }
 
+  generateGethFaucet (volumes) {
+    let faucetService = {
+      image: 'livepeer/testnet-services:faucet',
+      ports: [
+        '3333:8080'
+      ],
+      depends_on: this.hasGeth ? ['geth'] : [],
+      networks: {
+        testnet: {
+          aliases: [`faucet`]
+        }
+      },
+      command: [
+        '-network',
+        '54321',
+        '-provider',
+        'http://geth:8545',
+        '-keystore',
+        'keystore',
+        '-address',
+        '0161e041aad467a890839d5b08b138c1e6373072'
+      ],
+      restart: 'unless-stopped',
+    }
+
+    return needToCreateGethFaucet(this.config) ? faucetService : undefined
+  }
+
+  generateGethTxFiller (volumes) {
+    let txFillerService = {
+      image: 'livepeer/testnet-services:txfiller',
+      depends_on: this.hasGeth ? ['geth'] : [],
+      networks: {
+        testnet: {
+          aliases: ['tx-filler']
+        }
+      },
+      command: [
+        '-senderAddr',
+        '0161e041aad467a890839d5b08b138c1e6373072',
+        '-chainID',
+        '54321',
+        '-provider',
+        'http://geth:8545',
+        '-keystoreDir',
+        'keystore',
+        '-password',
+        'password.txt'
+      ],
+      restart: 'unless-stopped'
+    }
+    return needToCreateGethTxFiller(this.config) ? txFillerService : undefined
```

Comment:
okay I see what you did there, this way we don't need the if statement check before calling generate service.